packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -20,18 +20,15 @@
         mode="click"
       >
         <oc-list class="roleDropdownList">
-          <li
-            v-for="roleOption in availableRoleOptions"
-            :key="`role-dropdown-${roleOption.label.toLowerCase()}`"
-          >
+          <li v-for="roleOption in availableRoleOptions" :key="`role-dropdown-${roleOption.key}`">
             <oc-button
-              :id="`files-role-${roleOption.label.toLowerCase()}`"
+              :id="`files-role-${roleOption.key}`"
```

Comment:
Ahhh, that change causes the tests to fail because they look for the `id` including the role name. So either we switch to `data-test-id` for this, or revert it. What do you think?

Reply:
good catch, thank you! I took the third option / a variation of the first option: I changed the `id` prop again to use `roleOption.name` because that is a lower-case html safe attribute already. With labels you never know for sure, since they're meant for the human eye. That's why I wanted to remove it from the template for the id in the first place.